lib/test/karma.js

```diff
@@ -155,7 +166,9 @@ function karmaRun(reporters, bannerWidth) {
   var options = {
     configFile: undefined
   };
-  reporters = reporters || [];
+  if (reporters.constructor !== Array || reporters.length === 0) {
+    throw new Error('No reporters specified');
```

Comment:
This was done with the view that this file may be separated out into a library in its own right.
In angularity this can never happen, because when none are specified, the default is used (see `tasks/javascript.js`.